nextstrain_profiles/nextstrain-open/config.yaml

```diff
@@ -7,5 +7,4 @@ keep-going: False
 printshellcmds: True
 show-failed-logs: True
 restart-times: 2
-stats: stats.json
```

Comment:
Re: 4b2685cafb9c400374d2846b0c5c392aa74e1b1f

_(non-blocking)_

No more stats files? That's a bummer 😞 I've been using [Tom's vis tool](https://blab.github.io/snakemake-stats/) for these files to inspect performance of `augur filter` calls. We can still get the same numbers from per-rule benchmark files, but it's been nice to have a summary across all rules.

Reply:
I didn't really look into it (hence the "WIP" commit), I was just seeing the minimal amount of work to get the workflow running. The migration guide, somewhat comically, says "Unsupported?" so... maybe it's supported in some capacity? Please feel free to dive in here.

Reply:
Thanks, nothing wrong with your changes. Just wanted to rant 😄

Here's some discussion on its removal: https://github.com/snakemake/snakemake/issues/3523

The [reports](https://snakemake.readthedocs.io/en/v9.7.1/snakefiles/reporting.html) feature is a potential replacement, but I haven't used it.

Reply:
Since we are moving to v9 we should look into the reports feature, but from ~1 minute of scanning the docs it doesn't seem anywhere as nice. All in all we're not going to block the v9 upgrade on this.